website/docs/components/breadcrumb/partials/code/component-api.md

```diff
@@ -32,8 +32,11 @@ The Breadcrumb component is composed of three different parts, each with their o
   <C.Property @name="icon" @type="string">
     Use to show an icon. Any [icon](/icons/library) name is acceptable.
   </C.Property>
-  <C.Property @name="route/models/model/query">
-    These parameters are passed down as arguments to the `<LinkTo>` component.
+  <C.Property @name="route/models/model/query/current-when/replace">
+    Parameters that are passed down as arguments to the `<LinkTo>`/`<LinkToExternal>` components.
```

Comment:
you need to add `current-when/replace` to `<LinkTo>` for this to be true

Reply:
ok I merged your suggestion which did do that, so I'm going to resolve this comment. 👍